sdk-setup/systemd/sdk-webappstub.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Nothing to see here go away
+
+[Service]
+Type=simple
+ExecStart=/bin/sh -c "while true ; do echo 'HTTP/1.0 404 Not Found\nContent-Length: 68\n\nThe Sailfish Control Centre is gone, use sfdk or the SailfishOS IDE.\n\n' | nc -l -p 8080;done"
```

Comment:
Doesn't the echo require '-e'? ...hmmm "Sailfish SDK Control Centre" would be more correct.